src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+function conversion(value) {
+  return +value
+    .split('').slice(1).join('').split(',').join('');
+}
+
+addEventListener('click', e => {
+  const sort = document.querySelector('thead').children[0].children;
+  const cheak = document.querySelector('tfoot');
+  const i = [...sort].findIndex(
+    element => element.innerText === e.target.innerText
+    && !cheak.contains(e.target));
+
+  if (i === -1) {
+    return;
+  }
+
+  const listValue = document.querySelector('tbody');
```

Comment:
Consider using a more descriptive variable name than 'listValue'. For example, 'tableBody'.